src/resources/js/components/Hello.vue

```diff
@@ -1,12 +1,24 @@
 <template>
-    <div class="container">
-        <h3>Hello</h3>
-        <div class="text-center" style="margin: 20px 0px 20px 0px;">
-            <span class="text-secondary">Hello world</span>
-        </div>
-    </div>
+<div class="container">
+  <h3>Hello</h3>
+  <div class="text-center" style="margin: 20px 0px 20px 0px;">
+    <span class="text-secondary">Hello world</span>
+  </div>
+</div>
 </template>
 
 <script>
-    export default {}
+export default {
+  data() {
+    return {
+      user: {},
+    };
+  },
+  created() {
+    axios.get('/api/user/1')
+      .then(response => {
+        this.user = response.data.user;
+      });
+  },
+}
```

Comment:
Not sure about the utility of that code

Reply:
this is the exemple code for miragejs left by doran in #105 if i recall correctly